### ISSUE TITLE
Use SOCKS5 proxy with credentials

### ISSUE THREAD

**Issue by anatomism** [suuhm/unblock-proxy.sh#5]
I'm currently testing this out and it seems to work well. Thanks @suuhm for creating!

I'm looking to use a SOCKS5 proxy (proxychains) with credentials (user/pass), but it doesn't seem to work and then reading proxychains.conf it does say

```
# TODO: ONLY OPEN PROXY. 
# NO CREDS YET SUPPPORTED IN AUTOMATIC MODE!!
```

Do you have any pointers - I'm trying to make it work with IP Vanish

Thanks

**Comment by suuhm** [comment]
Yeah, its possible. So just add your ipvanish proxy to proxychains config like this:

`Socks5 ipvaniship port username pssword`

Hope this helps
